gm4_rituals/data/rituals/functions/altar/processes.mcfunction

```diff
@@ -0,0 +1,53 @@
+execute unless block ~1 ~-1 ~-1 minecraft:red_sandstone_stairs[half=top] run function rituals:altar/destroy
+execute unless block ~-1 ~-1 ~1 minecraft:red_sandstone_stairs[half=top] run function rituals:altar/destroy
+execute unless block ~-1 ~-1 ~-1 minecraft:red_sandstone_stairs[half=top] run function rituals:altar/destroy
+execute unless block ~1 ~ ~1 minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~1 ~ ~-1 minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~-1 ~ ~1 minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~-1 ~ ~-1 minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~1 ~1 ~1 minecraft:red_sandstone_wall run function rituals:altar/destroy
+execute unless block ~1 ~1 ~-1 minecraft:red_sandstone_wall run function rituals:altar/destroy
+execute unless block ~-1 ~1 ~1 minecraft:red_sandstone_wall run function rituals:altar/destroy
+execute unless block ~-1 ~1 ~-1 minecraft:red_sandstone_wall run function rituals:altar/destroy
+execute unless block ~1 ~2 ~1 minecraft:red_sandstone_slab run function rituals:altar/destroy
+execute unless block ~1 ~2 ~-1 minecraft:red_sandstone_slab run function rituals:altar/destroy
+execute unless block ~-1 ~2 ~1 minecraft:red_sandstone_slab run function rituals:altar/destroy
+execute unless block ~-1 ~2 ~-1 minecraft:red_sandstone_slab run function rituals:altar/destroy
+execute unless block ~1 ~2 ~ minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~-1 ~2 ~ minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~ ~2 ~1 minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~ ~2 ~-1 minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~1 ~3 ~ minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~-1 ~3 ~ minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~ ~3 ~1 minecraft:red_sandstone run function rituals:altar/destroy
+execute unless block ~ ~3 ~-1 minecraft:red_sandstone run function rituals:altar/destroy
+
+#water checks
+execute if block ~1 ~-1 ~ minecraft:red_sandstone_stairs[waterlogged=true] if block ~-1 ~-1 ~ minecraft:red_sandstone_stairs[waterlogged=true] if block ~ ~-1 ~1 minecraft:red_sandstone_stairs[waterlogged=true] if block ~ ~-1 ~-1 minecraft:red_sandstone_stairs[waterlogged=true] run tag @s add gm4_altar.water
+
+execute unless block ~1 ~-1 ~ minecraft:red_sandstone_stairs[waterlogged=true] run tag @s remove gm4_altar.water
+execute unless block ~-1 ~-1 ~ minecraft:red_sandstone_stairs[waterlogged=true] run tag @s remove gm4_altar.water
+execute unless block ~ ~-1 ~1 minecraft:red_sandstone_stairs[waterlogged=true] run tag @s remove gm4_altar.water
+execute unless block ~ ~-1 ~-1 minecraft:red_sandstone_stairs[waterlogged=true] run tag @s remove gm4_altar.water
```

Comment:
Same issue as the above block checks. Just remove the water tag before line 36 (#water checks). These 4 commands are unnecessary.